tests/test_manufactured.py

```diff
@@ -27,16 +55,17 @@ class Line1D(TestCase):
     Solution is u(x) = x.
 
     """
+
     e = ElementLineP1()
 
     def runTest(self):
-        m = MeshLine(np.linspace(0., 1.)).refined(2)
+        m = MeshLine(np.linspace(0.0, 1.0)).refined(2)
```

Comment:
I'm not sure, but it seems like you've run `pyblack` here. Do you think we should start using that project-wide?